{"error": null, "issues": [{"address": 2456, "code": "ayment(_level, msg.s", "debug": "timestamp: 0xa8795cd3fffffffffffffffffffffffffffffffffffffffffffffffffffffffe\nstorage_7: 0x57f25cd000000000000000000000000000000000000000000000000000000000\nstorage_0 +\nkeccac_1461501637330902918203684832716283019655932542975_&\n1461501637330902918203684832716283019655932542975_&\ncaller: 0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff\nstorage_5: 0x1\ncalldatasize_Smartex: 0x3\nstorage_keccac_0: 0x0\nstorage_105409183525425523237923285454331214386340807945685310246717412709691342439136: 0x0\ncallvalue: 0x0\n", "description": "A possible integer overflow exists in the function `fallback`.\nThe addition or multiplication may result in a value higher than the maximum representable integer.", "filename": "/temp/Smartex-0.sol", "function": "fallback", "lineno": 132, "title": "Integer Overflow ", "type": "Warning"}, {"address": 2640, "code": "elEvent(msg.sender, _level, now);\n  }\n\n  function findRefe", "debug": "storage_7: 0xe6089257c80ac0cbe60d5926ff262844c7005edbab82a162281118017d351c97\nstorage_keccac_0: 0x83ffb3569638cea3622d597040d78d7ab7ae5c8a2d8bbb012cad3fb961bad806\nstorage_0 +\nkeccac_1461501637330902918203684832716283019655932542975_&\n1461501637330902918203684832716283019655932542975_&\ncaller: 0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff\nstorage_5: 0x1\ncalldatasize_Smartex: 0x3\nstorage_105409183525425523237923285454331214386340807945685310246717412709691342439136: 0x0\ncallvalue: 0x0\n", "description": "A possible integer overflow exists in the function `fallback`.\nThe addition or multiplication may result in a value higher than the maximum representable integer.", "filename": "/temp/Smartex-0.sol", "function": "fallback", "lineno": 133, "title": "Integer Overflow ", "type": "Warning"}, {"address": 3482, "code": ".sender;\n    us", "debug": "storage_24465873643947496235832446106509767096567058095563226156125564318740882468607: 0xaa108000002bd3db7704854d60910b506d8900000000000000000000000051ac\nstorage_1: 0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff\nstorage_2 +\nkeccac_1461501637330902918203684832716283019655932542975_&\n1461501637330902918203684832716283019655932542975_&\n1461501637330902918203684832716283019655932542975_&\nUDiv(storage_keccac_storage_24465873643947496235832446106509767096567058095563226156125564318740882468607,\n_____256): 0x0\nstorage_6: 0x1\nstorage_5: 0x1\ncalldatasize_Smartex: 0x3\nstorage_0 +\nkeccac_1461501637330902918203684832716283019655932542975_&\n1461501637330902918203684832716283019655932542975_&\ncaller: 0x0\nstorage_105409183525425523237923285454331214386340807945685310246717412709691342439136: 0x0\ncallvalue: 0x0\n", "description": "A possible integer overflow exists in the function `fallback`.\nThe addition or multiplication may result in a value higher than the maximum representable integer.", "filename": "/temp/Smartex-0.sol", "function": "fallback", "lineno": 112, "title": "Integer Overflow ", "type": "Warning"}, {"address": 4803, "code": "if (users[referrals[i]].r", "debug": "The exception is triggered under the following conditions:\n\nstorage_24465873643947496235832446106509767096567058095563226156125564318740882468607: 0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff\nstorage_1: 0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff\nstorage_5: 0x1\ncalldatasize_Smartex: 0x3\nstorage_2 +\nkeccac_1461501637330902918203684832716283019655932542975_&\n1461501637330902918203684832716283019655932542975_&\n1461501637330902918203684832716283019655932542975_&\nUDiv(storage_keccac_storage_24465873643947496235832446106509767096567058095563226156125564318740882468607,\n_____256): 0x0\nstorage_6: 0x0\nstorage_0 +\nkeccac_1461501637330902918203684832716283019655932542975_&\n1461501637330902918203684832716283019655932542975_&\ncaller: 0x0\nstorage_105409183525425523237923285454331214386340807945685310246717412709691342439136: 0x0\ncallvalue: 0x0\n", "description": "A reachable exception (opcode 0xfe) has been detected. This can be caused by type errors, division by zero, out-of-bounds array access, or assert violations. This is acceptable in most situations. Note however that `assert()` should only be used to check invariants. Use `require()` for regular input checking. ", "filename": "/temp/Smartex-0.sol", "function": "fallback", "lineno": 148, "title": "Exception state", "type": "Informational"}, {"address": 4993, "code": "RRALS_LIMIT) {\n        re", "debug": "The exception is triggered under the following conditions:\n\nstorage_2 +\nkeccac_1461501637330902918203684832716283019655932542975_&\n1461501637330902918203684832716283019655932542975_&\n1461501637330902918203684832716283019655932542975_&\nUDiv(storage_keccac_storage_24465873643947496235832446106509767096567058095563226156125564318740882468607,\n_____256): 0x1\nstorage_24465873643947496235832446106509767096567058095563226156125564318740882468607: 0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff\nstorage_6: 0x1\nstorage_1: 0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff\nstorage_5: 0x1\ncalldatasize_Smartex: 0x3\nstorage_0 +\nkeccac_1461501637330902918203684832716283019655932542975_&\n1461501637330902918203684832716283019655932542975_&\ncaller: 0x0\nstorage_105409183525425523237923285454331214386340807945685310246717412709691342439136: 0x0\ncallvalue: 0x0\n", "description": "A reachable exception (opcode 0xfe) has been detected. This can be caused by type errors, division by zero, out-of-bounds array access, or assert violations. This is acceptable in most situations. Note however that `assert()` should only be used to check invariants. Use `require()` for regular input checking. ", "filename": "/temp/Smartex-0.sol", "function": "fallback", "lineno": 148, "title": "Exception state", "type": "Informational"}], "success": true}
